frontend/containers/dashboard/open-calls/table/cells/actions/component.tsx

```diff
@@ -36,15 +36,15 @@ export const CellActions: FC<CellActionsProps> = ({
       case 'launch':
         setConfirmLaunch(true);
         return;
-      // case 'edit':
-      //   router.push(
-      //     `${Paths.OpenCall}/${slug}/edit?returnPath=${encodeURIComponent(router.asPath)}`,
-      //     `${Paths.OpenCall}/${slug}/edit`
-      //   );
-      //   return;
-      // case 'preview':
-      //   router.push(`${Paths.OpenCall}/${slug}/preview`);
-      //   return;
+      case 'edit':
+        router.push(
+          `${Paths.OpenCall}/${slug}/edit?returnPath=${encodeURIComponent(router.asPath)}`,
+          `${Paths.OpenCall}/${slug}/edit`
+        );
+        return;
```

Comment:
We forgot to re-enable this on https://github.com/Vizzuality/heco-invest/pull/491